pkg/curatedpackages/packageclient.go

```diff
@@ -46,3 +59,74 @@ func convertBundleVersionToPackageVersion(bundleVersions []packagesv1.SourceVers
 	}
 	return versions
 }
+
+func (p *PackageClient) GeneratePackages() ([]packagesv1.Package, error) {
+	packageNameToPackage := p.getPackageNameToPackage()
+	var packages []packagesv1.Package
+	for _, v := range p.packages {
+		bundlePackage := packageNameToPackage[strings.ToLower(v)]
+		if bundlePackage.Name == "" {
+			fmt.Println(fmt.Errorf("unknown package %s", v).Error())
+			continue
+		}
+		name := customName + strings.ToLower(bundlePackage.Name)
```

Comment:
What is the reason for prepending `my-` here?

Reply:
That's the structure we are going with as of now. Whenever a user generates a package, the name of the package is called my-package. The idea is that they will change this if they need to.